pkg/sources/k8sevents/k8s_events.go

```diff
@@ -76,6 +76,10 @@ func (t *K8SEventsEventSource) StartFeed(trigger sources.EventTrigger, route str
 
 	// Just generate a random UUID as a subscriptionName
 	uuid, err := uuid.NewRandom()
+	if err != nil {
+		glog.Infof("Failed to create new random uuid: %v", err)
```

Comment:
It would be nice if this used `generateName` rather than making its own version.

Also, it looks like `subscriptionName` is just an alias for `generateName`. My guess is some copypasta.